examples/democoin/x/oracle/README.md

```diff
@@ -0,0 +1,53 @@
+# Oracle Module
+
+`x/oracle` provides a way to receive external information(real world price, events from other chains, etc.) with validators' vote. Each validator make transaction which contains those informations, and Oracle aggregates them until the supermajority signed on it. After then, Oracle sends the information to the actual module that processes the information, and prune the votes from the state.
```

Comment:
What if the validators disagree on the value?

Reply:
Slashing and confliction handling will be added later, but this is mvp for Oracle so the validators can only witness an event or do nothing. If (more than/) two conflicting events are passed Oracle process, module specific handlers must handle them.

Reply:
You might want to see if you can fold some the approaches for handling witness liveness from this research in a future revision.
https://arxiv.org/pdf/1804.05141.pdf